### PR TITLE
Purple notification bar + some code cleanup

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #867 

## Example of the screen a moderator would see after submitting an entry (entry gets auto approved right away)

https://network-pulse-staging-pr-884.herokuapp.com/entry/373?justPostedByUser=true (note that the query `justPostedByUser=true` should be clear after the page gets loaded)

screenshot-2017-12-4 finger heart love - mozilla network pulse

## Example of the screen a non moderator would see after submitting an entry (entry is in "pending" state)

https://network-pulse-staging-pr-884.herokuapp.com/submitted

screenshot-2017-12-4 thank you - mozilla network pulse

### CODE REVIEW

File: pages/add/submitted.jsx [hunk 1]
```diff
@@ -1,65 +1,20 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import qs from 'qs';
```
Comment (mmmavis) [inline pages/add/submitted.jsx]: This `Submitted` component should really be a stateless static component.

File: pages/entry.jsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import { Helmet } from "react-helmet";
+import { Link } from 'react-router-dom';
 import qs from 'qs';
 import LoadingNotice from '../components/loading-notice.jsx';
-import ProjectCardDetialed from '../components/project-card/project-card-detailed.jsx';
+import NotificationBar from '../components/notification-bar/notification-bar.jsx';
+import ProjectCardDetailed from '../components/project-card/project-card-detailed.jsx';
```
Comment (mmmavis) [inline pages/entry.jsx]: fixed typo in "detail"

File: components/notification-bar/notification-bar.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.notification-bar {
+  @extend .open-sans;
+
+  font-weight: 600;
+  background: rgba($pulse-purple, 0.2);
+  padding: 1rem;
```
Comment (gvn) [inline components/notification-bar/notification-bar.scss]: Is there a Bootstrap padding class that can be used instead?

Comment (mmmavis) [reply components/notification-bar/notification-bar.scss]: you mean do `@extend .p-3` here or add `p-3` as class to the notification component?

Comment (gvn) [reply components/notification-bar/notification-bar.scss]: Just adding a class to the component...

Comment (mmmavis) [reply components/notification-bar/notification-bar.scss]: @gvn this component will be used across the site. I wanna make sure it has consistent/fixed padding so I'd like keeping the padding rule here.

Comment (gvn) [reply components/notification-bar/notification-bar.scss]: If the class is in the React component definition it should be consistent everywhere it's used.

**Pull request merged**